### PR TITLE
Adapt documentation to terminal naming changes

### PR DESCRIPTION
**Pull request by aaannz**
Changes the terminal naming to reflect recent code changes in https://github.com/SUSE/spacewalk/issues/10504

### CODE REVIEW

File: modules/retail/pages/retail-terminal-names.adoc [hunk 1]
```diff
@@ -31,22 +23,43 @@ B002.TOSHIBA-6140100-41BA03X-c643
...
+The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.
+You can configure this value in the xref:salt:formula-branch-network.adoc[] settings for the branch server.
+You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[].
+
+The [systemitem]``Hostname`` is the plain hostname (without domain part) of the terminal.
+
+The [systemitem]``UniqueID`` is the first four characters of a generated machine identification number.
+You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branch-network.adoc[].
+
+=== Naming using the FQDN naming scheme
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
=== Naming by ``FQDN``
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 2]
```diff
@@ -31,22 +23,43 @@ B002.TOSHIBA-6140100-41BA03X-c643
...
+
+Terminal names that are derived from the Fully Qualified Domain Names  use this format:
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
Terminal names that are derived from the Fully Qualified Domain Names (FQDN) use this format:
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 3]
```diff
@@ -31,22 +23,43 @@ B002.TOSHIBA-6140100-41BA03X-c643
 ----
 
 
-The branch ID is the unique identifier for the branch server that the terminal is connected to.
-You can configure this value in the PXE formula settings for the branch server.
+The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.
+You can configure this value in the xref:salt:formula-branch-network.adoc[] settings for the branch server.
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can configure this value in the xref:salt:formula-branchnetwork.adoc[] settings for the branch server.
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 4]
```diff
@@ -31,22 +23,43 @@ B002.TOSHIBA-6140100-41BA03X-c643
...
+You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[].
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 5]
```diff
@@ -31,22 +23,43 @@ B002.TOSHIBA-6140100-41BA03X-c643
...
+For example:
 
-Terminal names that are derived from the hostname use this format:
+----
+B002.terminal-c643
+----
+
+
+The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.
+You can configure this value in the xref:salt:formula-branch-network.adoc[] settings for the branch server.
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can configure this value in the xref:salt:formula-branchnetwork.adoc[] settings for the branch server.
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 6]
```diff
@@ -31,22 +23,43 @@ B002.TOSHIBA-6140100-41BA03X-c643
...
+You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[].
+
+The [systemitem]``Hostname`` is the plain hostname (without domain part) of the terminal.
+
+The [systemitem]``UniqueID`` is the first four characters of a generated machine identification number.
+You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branch-network.adoc[].
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 7]
```diff
@@ -60,16 +73,13 @@ B002.terminal.example.com-c643
 
 
 The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.
-You can configure this value in the PXE formula settings for the branch server.
+You can configure this value in the xref:salt:formula-branch-network.adoc[Branch Network formula] settings for the branch server.
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can configure this value in the xref:salt:formula-branchnetwork.adoc[] settings for the branch server.
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 8]
```diff
@@ -60,16 +73,13 @@ B002.terminal.example.com-c643
...
+You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[Branch Network formula].
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

File: modules/retail/pages/retail-terminal-names.adoc [hunk 9]
```diff
@@ -60,16 +73,13 @@ B002.terminal.example.com-c643
...
 
 The [systemitem]``FQDN`` is the fully-qualified domain name of the terminal.
 
 The [systemitem]``UniqueID`` is the first four characters of a generated machine identification number.
-You can disable this behavior by toggling the [systemitem]``DISABLE_UNIQUE_SUFFIX`` parameter in the PXE formula settings.
-
-For more information about the PXE formula, see xref:salt:formula-pxe.adoc[].
-
-
+You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branch-network.adoc[Branch Network formula].
```
Comment (keichwa) [inline modules/retail/pages/retail-terminal-names.adoc]:
```suggestion
You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

File: modules/salt/pages/formula-branchnetwork.adoc [hunk 10]
```diff
@@ -62,5 +60,17 @@ Ensure you include DNS, TFTP, and FTP services.
 * Select the [guimenu]``bind`` DNS forwarder mode.
 * Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
 * Specify the working directory, and the directory owner and group.
+
+
+== Set up Branch Server Terminal Naming
+
+In this configuration it is required to fill at least [systemitem]``Branch Identification``. This identifies Branch Server in Retail subsystem and is also used to better organize terminals with their respective branch servers.
```
Comment (keichwa) [inline modules/salt/pages/formula-branchnetwork.adoc]:
```suggestion
In this configuration it is required to fill at least [systemitem]``Branch Identification``.
This identifies Branch Server in Retail subsystem and is also used to better organize terminals with their respective branch servers.
```

File: modules/salt/pages/formula-branchnetwork.adoc [hunk 11]
```diff
@@ -62,5 +60,17 @@ Ensure you include DNS, TFTP, and FTP services.
...
+
+.Procedure: Set up a Branch Server identification
```
Comment (keichwa) [inline modules/salt/pages/formula-branchnetwork.adoc]:
```suggestion
.Procedure: Setting up a Branch Server Identification
```

File: modules/salt/pages/formula-branchnetwork.adoc [hunk 12]
```diff
@@ -62,5 +60,17 @@ Ensure you include DNS, TFTP, and FTP services.
...
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. In the [guimenu]``Terminal Naming`` section, enter [systemitem]``Branch Identification`` string.
```
Comment (keichwa) [inline modules/salt/pages/formula-branchnetwork.adoc]:
```suggestion
. In the [guimenu]``Terminal Naming`` section, enter the [systemitem]``Branch Identification`` string.
```

**Comment by aaannz** [comment]
Thank you for the review, I applied your suggestions.

**Comment by aaannz** [comment]
I cannot merge this PR, so if all is ok with you, may I ask to merge this? This is part of the 4.0.5 being released this week.

**Comment by keichwa** [comment]
> I cannot merge this PR, so if all is ok with you, may I ask to merge this? This is part of the 4.0.5 being released this week.

Well done!  I completely forgot about the merge trouble.

**Pull request merged**